spec/lcm/integration/brick_runner.rb

```diff
@@ -8,8 +8,34 @@ def users_brick(opts)
       GoodData::Bricks::Pipeline.users_brick_pipeline.call script_params opts
     end
 
+    def release_brick(opts)
+      result = GoodData::Bricks::Pipeline.release_brick_pipeline.call script_params opts
+      pp result
+      master_ids = result[:results]['CreateSegmentMasters'].map { |r| r[:master_pid] }
+      fetch_projects(master_ids, opts) if opts[:client]
+    end
+
+    def provisioning_brick(opts)
+      result = GoodData::Bricks::Pipeline.provisioning_brick_pipeline.call script_params opts
+      pp result
+      client_pids = result[:params][:synchronize].first[:to].map(&:pid)
+      fetch_projects(client_pids, opts) if opts[:client]
+    end
+
+    def rollout_brick(opts)
```

Comment:
warning: Similar code found in brick_runner.rb:18, brick_runner.rb:25 (mass = 62) (flay)